card/lib/card/query/value.rb

```diff
@@ -34,8 +34,8 @@ def canonicalize_operator
 
       def sqlize v
         case v
-        when Query, SqlCond; v.to_sql
-        when Array;    "(" + v.flatten.collect {|x| sqlize(x)}.join(',') + ")"
+        when Query;  v.to_sql
+        when Array;  "(" + v.flatten.collect {|x| sqlize(x)}.join(',') + ")"
```

Comment:
Prefer `map` over `collect`.<br>Unnecessary spacing detected.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.